cypress/integration/ContributionForm_spec.js

```diff
@@ -23,23 +23,37 @@ describe('ContributionForm View', () => {
   })
 
   it('Should have a controlled input field for specialty whose value reflects the data typed into the form', () => {
-    cy.get('form input[name="specialty"]').type('trans health')
-      .get('form input[name="specialty"]').should('have.value', 'trans health')
+    cy.get('form input[name="specialties"]').type('trans health')
+      .get('form input[name="specialties"]').should('have.value', 'trans health')
   });
 
   it('Should have a controlled input field for insurance whose value reflects the data typed into the form', () => {
-    cy.get('form input[name="insurance"]').type('sliding scale')
-      .get('form input[name="insurance"]').should('have.value', 'sliding scale')
```

Comment:
Lets flesh out these tests for full user flow. 

Great initiative, over all, for consistently testing as you go.